src/main/java/org/apache/sling/distribution/journal/messages/PackageStatusMessage.java

```diff
@@ -47,7 +47,11 @@
         /**
          * The package has been imported
          */
-        IMPORTED(2);
+        IMPORTED(2),
+        /**
+         * The package invalidated the cache
+         */
+        INVALIDATED(3);
```

Comment:
I think that we should remove the `INVALIDATED` status and only use `IMPORTED` because the handling of `INVALIDATED` and `IMPORTED` should be the same in https://github.com/apache/sling-org-apache-sling-distribution-journal/pull/101

To be semantically correct, we should `s/IMPORTED/APPLIED` but we can't easily change that. I suggest to keep the `IMPORTED` constant but extend the JavaDoc with something along the lines of

```
The package has been applied
```